README.md

```diff
@@ -1 +1,19 @@
-# meta-realt
\ No newline at end of file
+# meta-realt
+To run app:
+  1. Go to server directory and install dependencies from file 'packege.json':
+  ```
+  yarn install
+  ```
+  2. Go to folder 'src', start server with the command:
+  ```
+  node index.js
+  ```
+  3. Go to root directory of project, install dependencies:
```

Comment:
It is possible only in a separate tab :) let's split the steps to two sections "Server setup", "FE App setup"